### DISCUSSION TITLE
Need to club at least 5 dependency update in single PR

### DISCUSSION THREAD

**Discussion by mhandroid** [renovatebot/renovate#31695] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Here is my configuration:
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "repositories": [
    "MobiCategories"
  ],
  "baseBranches": [
    "master"
  ],
  "packageRules": [
    {
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["minor"],
      "groupName": "Minor Updates",
      "groupSlug": "minor-lib",
      "minimumReleaseAge": "7d",
      "matchCurrentAge": "7d",
      "automerge": false,
      "prCreation": "not-pending",
      "enabled": true,
      "labels": ["minor-dependencies"]
    },
    {
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["patch"],
      "groupName": "Patch Updates",
      "groupSlug": "patch-lib",
      "minimumReleaseAge": "2d",
      "matchCurrentAge": "2d",
      "automerge": false,
      "prCreation": "not-pending",
      "enabled": true,
      "labels": ["patch-dependencies"]
    },
    {
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["major"],
      "groupName": "Major Updates",
      "groupSlug": "major-lib",
      "minimumReleaseAge": "14d",
      "matchCurrentAge": "14d",
      "automerge": false,
      "prCreation": "not-pending",
      "enabled": true,
      "labels": ["dependencies", "breaking"]
    }
  ],
  "prHourlyLimit": 3,
  "prConcurrentLimit": 3,
  "branchConcurrentLimit": 3,
  "stabilityDays": 7,
  "schedule":[
    "at any time"
  ],
  "vulnerabilityAlerts": {
    "enabled": true
  },
  "assigneesFromCodeOwners": true
}

1. I would like to create a single PR which can have at least 5 dependency update instead of a single PR for single dependency update. Limit must for example 5 must be configurable. 
2. How to exclude certain package which I don't want renovate to look. those dependencies are critical like payment related etc.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
1. There's no direct config option to limit the number of deps updated in a PR. You will have to try multiple combinations using packagrRules, groupName and available matchers till one works for you.

2. To ignore certaind deps use the config option `ignoreDeps`
